### PR TITLE
249 add way to read torque and speed limit

### PR DESCRIPTION
**Pull request by glannuzel**
- set torque and speed limits at actuators and part level
- read torque and speed limits at actuators level, in a dict composed of all motors

### CODE REVIEW

File: src/reachy2_sdk/orbita/orbita2d.py [hunk 1]
```diff
@@ -134,6 +134,41 @@ def send_goal_positions(self) -> None:
         self._outgoing_goal_positions = {}
         self._stub.SendCommand(command)
 
+    def set_speed_limits(self, speed_limit: float | int) -> None:
+        """Set a speed_limit as a percentage of the max speed on all motors of the actuator"""
+        super().set_speed_limits(speed_limit)
+        speed_limit = speed_limit / 100.0
```
Comment (FabienDanieau) [inline src/reachy2_sdk/orbita/orbita2d.py:140]: This is also used in orbita3d. move to super() ?

File: src/reachy2_sdk/orbita/orbita.py [hunk 2]
```diff
@@ -91,15 +92,15 @@ def __repr__(self) -> str:
     #     else:
     #         raise ValueError("pid should be of type Tuple[float, float, float]")
 
-    def get_speed_limit(self) -> Dict[str, float]:
+    def get_speed_limits(self) -> Dict[str, float]:
         """Get speed_limit of all motors of the actuator"""
```
Comment (FabienDanieau) [inline src/reachy2_sdk/orbita/orbita.py:96]: we should add in the comment the unit (percentage of the max speed right?)

File: src/reachy2_sdk/orbita/orbita_motor.py [hunk 3]
```diff
@@ -59,8 +46,7 @@ def pid(self) -> PIDGains:
 
     def _update_with(self, new_state: Dict[str, FloatValue]) -> None:
         self._temperature = new_state["temperature"].value
-        self._speed_limit = new_state["speed_limit"].value
-        self._torque_limit = new_state["torque_limit"].value
+        self._speed_limit = new_state["speed_limit"].value * 100
```
Comment (FabienDanieau) [inline src/reachy2_sdk/orbita/orbita_motor.py:49]: add to comment saying that received values are between 0 and 1

**Review by FabienDanieau — APPROVED** [review]
Just minor comments. Looks good, tested with fake robot

File: src/reachy2_sdk/orbita/orbita2d.py (hunk 1 above)
Comment (glannuzel) [reply src/reachy2_sdk/orbita/orbita2d.py:140]: Didn't manage to properly modify this variable in the super()

File: src/reachy2_sdk/orbita/orbita.py (hunk 2 above)
Comment (glannuzel) [reply src/reachy2_sdk/orbita/orbita.py]: done

**Pull request merged**